java/ingest/ingest-starter/src/main/java/sleeper/ingest/starter/RunTasks.java

```diff
@@ -83,18 +84,24 @@ public void run() throws InterruptedException {
             return;
         }
 
-        // Find out number of running tasks
-        int numRunningTasks = CommonJobUtils.getNumRunningTasks(clusterName, ecsClient);
-        LOGGER.info("Number of running tasks is {}" + numRunningTasks);
+        // Find out number of pending and running tasks
+        int numRunningAndPendingTasks = -1;
+        try {
+            numRunningAndPendingTasks = CommonJobUtils.getNumPendingAndRunningTasks(clusterName, ecsClient);
+        } catch (DescribeClusterException e) {
+            LOGGER.error("DescribeClusterException when retrieving number of pending and running tasks for cluster " + clusterName, e);
```

Comment:
This can use substitution rather than string concatenation, ie. error("...for cluster {}", clusterName, e).